### PR TITLE
Added buildifier to sigh lint

### PR DESCRIPTION
**Pull request by csilvestrini**
This now lints all BUILD and WORKSPACE files with buildifier. Run `./tools/sigh lint --fix` to fix them automatically.

Also fixed WORKSPACE file, which had lint errors.

This is important for importing into google.

### CODE REVIEW

File: tools/sigh.ts [hunk 1]
```diff
@@ -537,6 +537,21 @@ function lint(args: string[]): boolean {
...
+  const options = minimist(args, {
+    boolean: ['fix'],
+  });
+
+  const modeOpts = options.fix ? ['--lint=fix', '--mode=fix'] : ['--lint=warn', '--mode=check'];
+
+  return saneSpawnSync('find', [
+    '.',
+    '-name', 'BUILD', '-o', '-name', 'BUILD.bazel', '-o', '-name', 'WORKSPACE', '-o', '-name', '*.bzl',
+    '-exec', 'node_modules/@bazel/buildifier/buildifier.js', ...modeOpts, '--warnings=-module-docstring,-bzl-visibility', '{}', '\\;'
```
Comment (piotrswigon) [inline tools/sigh.ts]: maybe `npx buildifier` would be sufficient?

File: tools/sigh.ts [hunk 2]
```diff
@@ -537,6 +537,21 @@ function lint(args: string[]): boolean {
...
+function buildifier(args: string[]): boolean {
+  const options = minimist(args, {
+    boolean: ['fix'],
+  });
+
+  const modeOpts = options.fix ? ['--lint=fix', '--mode=fix'] : ['--lint=warn', '--mode=check'];
+
+  return saneSpawnSync('find', [
+    '.',
+    '-name', 'BUILD', '-o', '-name', 'BUILD.bazel', '-o', '-name', 'WORKSPACE', '-o', '-name', '*.bzl',
```
Comment (piotrswigon) [inline tools/sigh.ts]: Ugh... second thought: Will this work on Windows?

Comment (csilvestrini) [reply tools/sigh.ts]: Sigh!

Comment (csilvestrini) [reply tools/sigh.ts]: Done.

File: tools/sigh.ts (hunk 1 above)
Comment (csilvestrini) [reply tools/sigh.ts]: Thanks, that works!

**Pull request merged**